doc/modules/clustering.rst

```diff
@@ -1388,3 +1388,29 @@ Drawbacks
   concepts of clusters, such as density based clusters like those obtained
   through DBSCAN.
 
+.. _contingency_matrix:
+
+Contingency Matrix
+----------------------
+
+Contingency matrix is a format describing the relationship between discrete-valued random variables.
```

Comment:
This sentence is very abstract and hard to understand. Maybe make it a bit more concrete with talking about cluster labels.